theano/tensor/basic.py

```diff
@@ -3963,16 +3963,78 @@ def shape_padright(t, n_ones=1):
 
 
 @constructor
-def stack(*tensors):
+def shape_padaxis(t, axis):
+    """Reshape `t` by adding 1 at the dimension `axis`.
```

Comment:
This need an example. It is not clear on which side of the axis the 1 is added. Both option are possible.